include/preferences.hpp

```diff
@@ -25,41 +25,39 @@ class QScrollArea;
 class QPushButton;
 
 class QPxPreferences : public QDialog {
-	Q_OBJECT
+  Q_OBJECT
 
 public:
-	QPxPreferences(QPxSettings *iset, QWidget *p=0, Qt::WindowFlags f=0);
-	virtual ~QPxPreferences();
+  QPxPreferences(QPxSettings *iset, QWidget *p = 0, Qt::WindowFlags f = 0);
```

Comment:
Codacy found an issue: [Class 'QPxPreferences' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/manual/jeremiahwilson777/qpxtool/pullRequest?prid=6191247)